.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>CreditCardValidatorProject</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+	</natures>
+</projectDescription>
```

Comment:
**Suggested**: 
files: .project, .classpath, .settings folder should be added to **.gitignore file**

**Explanation**:
These are Eclipse IDE files describing project settings.
Project was probably generated by particular IDE.
Some other members of the team might use different IDE and have different files generated.
Let the project setup be independent from the IDE particular developer is using.

I would recommend to setup a build of the project 
using standard build tool like **Maven** or **Gradle**.
Then you can add to repository file with information about the project and 
configuration details - e.g. pom.xml in case of Maven